py3status/modules/spaceapi.py

```diff
@@ -56,6 +61,7 @@ def check(self):
             reader = codecs.getreader("utf-8")
             data = json.load(reader(json_file))
             json_file.close()
+            self.url = data['url']
```

Comment:
> why do you redefine the url configuration parameter here?

You are right it should be `self._url` or something like that.  I was looking at the patch more than the whole file